scipy/stats/tests/test_qmc.py

```diff
@@ -445,63 +457,50 @@ class TestHalton(QMCEngineTests):
 class TestLHS(QMCEngineTests):
     qmce = qmc.LatinHypercube
     can_scramble = False
-    unscramble_nd = np.array([[0.73412877, 0.50416027],
-                              [0.5924405, 0.51284543],
-                              [0.57790629, 0.70797228],
-                              [0.44357794, 0.64496811],
-                              [0.23461223, 0.55712172],
-                              [0.45337347, 0.4440004],
-                              [0.73381992, 0.01751516],
-                              [0.52245145, 0.33099331]])
+
+    scramble = [True, False]
+    ids = ["Scrambled", "Unscrambled"]
 
     def test_continuing(self, *args):
         pytest.skip("Not applicable: not a sequence.")
 
     def test_fast_forward(self, *args):
         pytest.skip("Not applicable: not a sequence.")
 
-    def test_sample_centered(self):
-        engine = self.engine(d=2, scramble=False, centered=True)
-        sample = engine.random(n=5)
-        out = np.array([[0.3, 0.5],
-                        [0.5, 0.3],
-                        [0.1, 0.7],
-                        [0.7, 0.7],
-                        [0.7, 0.1]])
-        assert_almost_equal(sample, out, decimal=1)
+    def test_sample(self, *args):
+        pytest.skip("Not applicable: the value of reference sample is implementation dependent.")
 
+    def test_continuing(self, *args):
+        pytest.skip("Not applicable: the value of reference sample is implementation dependent.")
```

Comment:
The test is already skipped with a better error message.
```suggestion
```

Reply:
Where is it skipped? I cannot recognize where it is...

Reply:
When opening the file from your PR, I can see at the beginning:

```python
    def test_continuing(self, *args):
        pytest.skip("Not applicable: not a sequence.")
```

Reply:
https://github.com/scipy/scipy/blob/486aa7d9af36a1d0a5264e5290398c22ebcbda6d/scipy/stats/tests/test_qmc.py#L464